build.gradle

```diff
@@ -563,6 +565,8 @@ def commonIntegTestClusters(OpenSearchCluster cluster, _numNodes){
 
 testClusters.integTest {
     commonIntegTestClusters(it, _numNodes)
+    // Forcing optimistic search for testing
+    systemProperty 'mem_opt_srch.force_reenter', 'true'
```

Comment:
do we need this ?

Reply:
If you want to run search for memory optimized cases then lets create another gradle task and also a new CI that runs that task.